packages/docusaurus-theme-classic/src/theme/CodeBlock/CopyButton/index.tsx

```diff
@@ -40,7 +40,6 @@ export default function CopyButton({code}: Props): JSX.Element {
               description: 'The ARIA label for copy code blocks button',
             })
       }
-      // @todo: check it again later
```

Comment:
Let's settle on using `// TODO` in the future, because only this can be tracked by my vscode extension:

<img width="471" alt="image" src="https://user-images.githubusercontent.com/55398995/159956927-fa935c44-8be6-4672-9845-5d218769fe7e.png">

<img width="471" alt="image" src="https://user-images.githubusercontent.com/55398995/159957004-50b1d527-97d9-4c0f-9e75-7126e919ffbd.png">